openapi/resource-server.yaml

```diff
@@ -253,7 +253,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/outgoing-payment'
+                $ref: '#/components/schemas/outgoing-payment-with-spent-amounts'
```

Comment:
Note that this type is only used for the `POST`. I believe this is what we want based on the task description and since we are already getting these amounts in the create, but I figured I should confirm this direction.

I can imagine it would be unnecessarily expensive to do it in the `GET /outgoing-payments/{id}` and especially `GET /outgoing-payments` and presumably the create is sufficient for whatever is consuming this information?

Reply:
this was confirmed in a discussion about this task